src/frontend/src/components/public/PIAFormTabs/accuracyRetention/index.tsx

```diff
@@ -160,26 +160,34 @@ export const AccuracyCorrectionAndRetention = () => {
                 </p>
               </>
             ) : (
-              <h4>
-                <a
-                  href={
+              <>
+                <h4>
+                  {
                     Messages.FormElements.RequestCorrection.haveProcessinPlace
-                      .HelperText.LinkHref
+                      .Question.en
                   }
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
+                </h4>
+                <h4>
+                  <a
+                    href={
+                      Messages.FormElements.RequestCorrection.haveProcessinPlace
+                        .HelperText.LinkHref
+                    }
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    {
+                      Messages.FormElements.RequestCorrection.haveProcessinPlace
+                        .HelperText.LinkText.en
+                    }
+                    <FontAwesomeIcon icon={faUpRightFromSquare} />
+                  </a>
```

Comment:
This helper text should only be shown in the edit mode.

Reply:
remove it